protocols/identify/src/behaviour.rs

```diff
@@ -183,7 +183,7 @@ impl Behaviour {
     {
         for p in peers {
             if !self.connected.contains_key(&p) {
-                log::debug!("Not pushing to {p} because we are not connected");
+                tracing::debug!("Not pushing to {p} because we are not connected");
```

Comment:
Not a must but if you are feeling like it, statements like these could be re-written to take advantage of structured logging, e.g.:

```suggestion
                tracing::debug!(peer = %p, "Cannot push to a peer we are not connected to");
```

Reply:
There are likely dozens of such opportunities in the codebase :)

Let me know how far you want to take this!

Reply:
just to make sure I'm understanding this correctly

would we want something like this
`tracing::info!("Discovered peer {} at {}", peer, address);`
to be converted to
`tracing::info!(peer=%peer,  "Discovered peer at {}", address);`

Reply:
Or even better:

```rust
tracing::info!(%peer, %address, "Discovered peer");
```